clusterloader2/testing/l4ilb/config.yaml

```diff
@@ -0,0 +1,153 @@
+#Constants
+{{$LARGE_BACKEND_SIZE := 300}}
+{{$MEDIUM_BACKEND_SIZE := 150}}
+{{$SMALL_BACKEND_SIZE := 10}}
+{{$LARGE_BACKEND_LABEL := "ilblarge"}}
+{{$MEDIUM_BACKEND_LABEL := "ilbmedium"}}
+{{$SMALL_BACKEND_LABEL := "ilbsmall"}}
+# adding a fixed value for first version of the test, rate of pod creation not a concern yet.
+{{$ilbQPS := 500}}
+{{$namespaces := 1}}
+#Test
+name: ilbload
+automanagedNamespaces: {{$namespaces}}
+tuningSets:
+- name: ILBConstantQPS
+  qpsLoad:
+    averageQps: {{$ilbQPS}}
+steps:
+# Create ILBs
+- measurements:
+  - Identifier: ServiceCreationLatencyLarge
+    Method: ServiceCreationLatency
+    Params:
+      action: start
+      labelSelector: size = ilblarge
```

Comment:
nit: `labelSelector: size = {{$LARGE_BACKEND_LABEL}}`
Same for medium and small

Reply:
done